### ISSUE TITLE
Inductor to convergence training accuracy consistently worse than eager

### ISSUE THREAD

**Issue by msaroufim** [pytorch/torchdynamo#1905]
Repro: https://github.com/sgugger/torchdynamo-tests/issues/5

![image](https://user-images.githubusercontent.com/3282513/203203854-ae11ab97-054a-4cb3-bb22-d78334bdf5e9.png)

Full data: https://docs.google.com/spreadsheets/d/1a-0lEAGRnfOqOzHgVafNDAU6ishsScWaYetXY-YsCv4/edit#gid=0

**Comment by msaroufim** [comment]
Data is bad will reopen another issue if this issue persists
